lesson15_homework_sims/homework15.py

```diff
@@ -0,0 +1,131 @@
+from random import randint
+
+
+class NotEnoughMoney(Exception):
+    def __init__(self, some_to_buy):
+        self.some_to_buy = some_to_buy
+
+    def __str__(self):
+        return f"Not enough money to buy {self.some_to_buy}"
+
+
+class ChattelFirst(Exception):
+    def __str__(self):
+        return "You must buy chattel before buy estate"
+
+
+class Property:
+    def __init__(self, price, name):
+        self.__price = price
+        self.__name = name
+
+    @property
+    def price(self):
+        return self.__price
+
+    def __repr__(self):
+        return f"{self.__name} - {self.price}"
+
+
+class Chattel(Property):
+    @property
+    def type_of_property(self):
+        return 'chattel'
```

Comment:
try to look `Enum` and use it here and where you check this type.. I mean `Human.is_chattel_in_property`

Reply:
do you mean replace this:


    def sell(self, some_property):
        for i in range(len(self.property) - 1):
            if some_property == self.property[i]:
                self.property.pop(i)
                self.money += some_property.price
                print(f'You have sell {some_property}')
                return
            else:
                print(f'You do not have {some_property}')

for this:

def sell(self, some_property):
    for prop in enumerate(self.property):
        if some_property == prop[1]:
            self.property.pop(prop[0])
            self.money += some_property.price
            print(f'You have sell {some_property}')
            return
        else:
            print(f'You do not have {some_property}')